src/app/components/WalletPopUp/WalletPopUp.tsx

```diff
@@ -25,16 +19,16 @@ const WalletPopUp = () => {
     const walletStore: WalletStore = stores.Wallet;
 
     const changeWallet = () => {
-        applicationStore.setShowPopUp(false);
+        applicationStore.setShowWalletPopUp(false);
         walletStore.onboardSdk.walletSelect();
     };
 
     const closePopUp = () => {
-        applicationStore.setShowPopUp(false);
+        applicationStore.setShowWalletPopUp(false);
     };
 
     const openWalletAddress = () => {
-        window.open(`https://goerli.etherscan.io/address/${walletStore.accountAddress}`);
+        window.open(config.links.ETHER_SCAN_LINK + walletStore.accountAddress);
```

Comment:
There's common rule to use `${}...${}` concatenation when possible instead of `+` when you work with strings.